src/Integration/Language.cs

```diff
@@ -38,13 +38,13 @@ namespace SonarLint.VisualStudio.Integration
     /// This class is safe for use as a key in collection classes. E.g., <seealso cref="IDictionary{TKey, TValue}"/>.
     /// </para>
     /// </remarks>
-    [DebuggerDisplay("{Name} (ID: {Id}, ProjectType: {ProjectType}, IsSupported: {IsSupported})")]
+    [DebuggerDisplay("{Name} (ID: {Id}, IsSupported: {IsSupported})")]
     [TypeConverter(typeof(LanguageConverter))]
     public sealed class Language : IEquatable<Language>
     {
         public readonly static Language Unknown = new Language();
-        public readonly static Language CSharp = new Language("CSharp", Strings.CSharpLanguageName, ProjectSystemHelper.CSharpProjectKind);
-        public readonly static Language VBNET = new Language("VB", Strings.VBNetLanguageName, ProjectSystemHelper.VbProjectKind);
+        public readonly static Language CSharp = new Language("CSharp", Strings.CSharpLanguageName);
```

Comment:
The _Language_ class used to have a one-one mapping from language -> project type. I've removed this.